nex/devrunner.go

```diff
@@ -49,6 +49,14 @@ func init() {
 // Attempts to "deploy a file" by finding a suitable target and publishing the workload to an ad-hoc created bucket
 // and using default issuer and publisher keys stored in ~/.nex. This should be as easy as typing "nex devrun ./amazingapp env1=foo env2=bar"
 func RunDevWorkload(ctx context.Context, logger *slog.Logger) error {
+	if RunOpts.WorkloadType == "elf" || RunOpts.WorkloadType == "native" || true {
```

Comment:
Why is the "or true" on here? Doesn't that make this code block always run?

Reply:
it does...because WorkloadType is being determined after the file is uploaded, which isnt going to work here.  ideally, we will required workload for devrun (and default it to elf/native)....i can add that logic